src/CMakeLists.txt

```diff
@@ -6,14 +6,7 @@ set(CMAKE_CXX_STANDARD 17)
 
 # Find boost required components
 set(BOOST_COMPONENTS)
-list(APPEND BOOST_COMPONENTS program_options system filesystem)
-
-if (APPLE)
-    list(APPEND BOOST_COMPONENTS thread-mt log-mt log_setup-mt)
-else()
-    list(APPEND BOOST_COMPONENTS thread log log_setup)
-endif()
-
+list(APPEND BOOST_COMPONENTS program_options system filesystem thread log log_setup)
```

Comment:
I think we shouldn't respect `-mt` prefix anymore. It was an artifact of `brew` installation on Mac, and at the moment the brew boost version is not compatible with our code anyway, so we have to install boost manually on mac - in that case no `-mt` suffix will be present

Reply:
Agree